source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -4,135 +4,382 @@ Sort Results
 
 .. default-domain:: mongodb
 
-Use ``sort()`` to change the order in which read operations return
-documents. The ``sort()`` instance method tells MongoDB to order returned documents by the
-values of one or more fields in a certain direction. To sort returned
-documents by a field in ascending (lowest first) order, use the static method ``Sorts.ascending()``, 
-specifying the field on which to sort. To sort in descending (greatest first) order instead, 
-use the static method ``Sorts.descending()``, specifying the field on which to sort. For example,
-you can sort on ``id`` or ``name`` fields. If you do not specify a sort, MongoDB does not 
-guarantee the order of query results.
-
-Follow the instructions in the examples below to insert data into
-a collection and perform a sort on the results of a query.
-Consider a collection containing documents that describe books. To
-insert this data into a collection, run the following operation:
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, we show you how to **sort** your results.
+
+Sorting orders the documents returned from your query by your specified 
+**sort criteria**. Sort criteria are the rules you give MongoDB that describe
+how you would like your data ordered. Some examples of sort criteria are:
```

Comment:
```suggestion
how you would like your data to be ordered. Some examples of sort criteria are:
```
Or:
```suggestion
how you would like to order your data. Some examples of sort criteria are:
```

Reply:
Thank you for the note. Would you consider this a style edit or a grammar edit? Taking the suggestion, however both sentences sound correct to my ear.